icon_validator/rules/plugin_validators/version_bump_validator.py

```diff
@@ -266,10 +276,14 @@ def check_major_version_increment_needed(self, remote, local):
                                       "being numbers. No special characters or spaces allowed. "
                                       "Versions start at 1.0.0, see https://semver.org/ for more information.")
 
-    def check_minor_version_increment_needed(self, remote, local):
+    def check_minor_version_increment_needed(self, remote: dict, local: dict):
+        # input: complete spec dictionary
+        # Checks to see if version is valid sem-ver, and if we already bumped minor version
         local_version = local["version"].split('.')
         remote_version = remote["version"].split('.')
         if len(local_version) == 3 and len(remote_version) == 3:
+            VersionBumpValidator.modify_version_array(local_version)
+            VersionBumpValidator.modify_version_array(remote_version)
```

Comment:
Did you mean to assign these function calls to variables? As it stands right now, these will not mutate `local_version` or `remote_version` so the semver with metadata will still pose an issue.

Reply:
I did, fixed in next commit